suricata/update/main.py

```diff
@@ -458,33 +250,14 @@ def load_filters(filename):
 
 def load_drop_filters(filename):
     
-    matchers = load_matchers(filename)
+    match = matchers.load_matchers(filename)
     filters = []
 
-    for matcher in matchers:
+    for matcher in match:
```

Comment:
`match` is already defined as a function in `DropRuleFilter` class. I think, changing variable name like this might have a conflict.

Reply:
@vidushi4 The variable name was "matchers" before which was conflicting with new matchers file as it was creating "Referenced before assignment " that is why I changed it. Also, it is a local variable, Therefore, it would not affect other functions. It was the most appropriate name that I could suggest . If you have something better, do suggest, I will update it while creating the new PR.

Reply:
Yes I did notice all of that. But all I was suggesting was that same names can create confusion if not for conflict. Let's wait for the mentors' reviews. This was just my opinion. Cheers!